x/staking/migrations/v5/store.go

```diff
@@ -90,3 +92,47 @@ func MigrateHistoricalInfoKeys(ctx sdk.Context, store storetypes.KVStore, index
 
 	return nil
 }
+
+// migrateParams will set the params to store from legacySubspace
+func migrateParams(ctx sdk.Context, store storetypes.KVStore, cdc codec.BinaryCodec) error {
+
+	// Get the params from the store
+	params, err := GetParams(ctx, store, cdc)
+	if err != nil {
+		return err
+	}
+
+	params.MaxCommissionRate = types.DefaultMaxCommissionRate
```

Comment:
should the migration force downgrade the validators commission rate if they are greater than the max commission rate?

Reply:
existing validators stay above. when they edit it goes below